app/assets/stylesheets/_globals.sass

```diff
@@ -7,6 +7,11 @@ html, body, .container, .app
 header
   position: fixed
   z-index: 5
+  width: 100%
+  top: 0
+  left: 0
+  background-color: $application-background-color
+  box-shadow: 0px -1px 14px 1px #c3c3c2
   .brand-logo
```

Comment:
Ik vind de uitlijning van het logo niet zo mooi, misschien kan hij beter verticaal in het midden?
![image](https://cloud.githubusercontent.com/assets/8157262/14421176/68b2102c-ffd0-11e5-9e5e-c662d2c8b5fe.png)